content/javascript/concepts/strings/terms/padEnd/padEnd.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.padEnd()'
+Description: 'Pads a string with a given string several times until it reaches the given length.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, the **padEnd()** method is used with javascript strings to pad the end of a string with another string, until it reaches a specified length. It was introduced with ECMAScript 2017.
+
+## Syntax
+
+```pseudo
+string.padEnd(length, padStr)
+```
+
+The string is a given variable containing a sentence, the length parameter is the value the string needs to reach and padStr is the string character to add to the given string until it reaches the desired length. Note if you wish to use this method with a nukber, you will need to convert to a string first.
```

Comment:
``` suggestion
The string is a given variable containing a sentence, the length parameter is the value the string needs to reach and padStr is the string character to add to the given string until it reaches the desired length. >**Note:** If you wish to use this method with a number, you must first convert to a string.
```
At the time of writing entries, the note should always be written using the blockquote(>) and also use an asterisk symbol like this(**). To understand it properly, you can get more information about this [at](https://github.com/Codecademy/docs/tree/main/documentation).